src/ui/components/search/searchcomponent.js

```diff
@@ -487,7 +498,12 @@ export default class SearchComponent extends Component {
     // serialized and submitted.
     if (typeof this.redirectUrl === 'string') {
       if (this._allowEmptySearch || query) {
-        window.location.href = this.redirectUrl + '?' + params.toString();
+        const newUrl = this.redirectUrl + '?' + params.toString();
+        if (this.redirectUrlTarget) {
+          window.open(newUrl, this.redirectUrlTarget) || (window.location.href = newUrl);
```

Comment:
would it be possible to default redirectUrlTarget to the current window, and not have this if check?